src/middleware/update-check.ts

```diff
@@ -13,7 +13,7 @@ export const updateCheck = async (): Promise<void> => {
 
     if (firefoxVersion && version !== firefoxVersion)
       log.warning(
-        `Latest version of Firefox (${version}) does not match frozen version (${firefoxVersion}).`
+        `Latest version of Firefox (${version}) does not match frozen version (${firefoxVersion}). Update Firefox with the command |gluon update|.`
```

Comment:
Please use `${bin_name}` instead of `gluon`

Reply:
Resolved in newest commit

Reply:
This doesn't seem to have been changed here

Reply:
Oh, I changed the wrong file.

Reply:
This should be fixed now